2-mandatory/1-writers.js

```diff
@@ -91,28 +96,28 @@ function logAliveWritersInTheirForties() {
 */
 
 test("exercise 1", () => expectFunctionToLog(logAllWriters, [
-  "Hi, my name is Virginia Woolf. I am 59 years old, and work as a writer.",
-  "Hi, my name is Zadie Smith. I am 40 years old, and work as a writer.",
-  "Hi, my name is Jane Austen. I am 41 years old, and work as a writer.",
-  "Hi, my name is Bell Hooks. I am 63 years old, and work as a writer.",
-  "Hi, my name is Yukiko Motoya. I am 49 years old, and work as a writer."
+    "Hi, my name is Virginia Woolf. I am 59 years old, and work as a writer.",
```

Comment:
Also I know it can be anoying to look at tests not indented right or there are other visual issues with it, but I dont recommend fixing the tests as it will waste too much time and all you will be doing is looking for 5 seconds max everytime you need to confirm the output or something like that.